### ISSUE TITLE
Documentation: Screen reader fails announcing about the hint (Press Ctrl K) to invoke the search button.

### ISSUE THREAD

**Issue by msftedad** [algolia/docsearch#2033]
## Description
User is not able to know the provided hint.
User with visual impairment who rely on screen reader will not be known about the short key to invoke the search button.

## Steps to reproduce

Open https://docusaurus.io/docs URL.
Navigate to search button and invoke it.
Listen to screen reader announcement.

**Live reproduction:**

 https://codesandbox.io/s/github/algolia/docsearch/tree/main/examples/demo](https://docusaurus.io/docs)

## Expected behavior
Screen reader should announce hint (Press Ctrl K) ti invoke the search button.

## Environment

OS Ver: 25324.1011
Edge Ver: 112.0.1722.23 (Official build) dev (64-bit)
URL: [FAST](https://www.fast.design/)
AT: Screen reader.
DOC Search version: 3.1

![Dev Tools](https://github.com/algolia/docsearch/assets/93735775/f4aa29af-f07c-43b7-b440-662e738d51d2)

https://github.com/algolia/docsearch/assets/93735775/6ac9df23-4e37-4775-b2c0-9627969b816e

https://github.com/algolia/docsearch/assets/93735775/070318a3-ab64-4b91-9d8a-2e39c344300e

**Comment by msftedad** [comment]
Hi team, Any update on this issue?
